MergeCalendarsTogether.gs

```diff
@@ -324,15 +368,17 @@ function MergeCalendars (calendars) {
         requests: calendarRequests,
       });
       if (!result.getResponseCode || result.getResponseCode() !== 200) {
-        console.log(result)
+        log.info(result)
+      } else {
+        log.debug('BatchRequest result: ', result.toString(), '; ', result.getResponseCode() )
```

Comment:
If you look at Batch Request, it only uses the _enhancedDo_ func when there are 100 or more simultaneous requests, and this only logged anything if _enhancedDo_ was used. As a result, we were not logging any failures for smaller batches.  This means if verbose logging is enabled, you can now see the batched 403s returning. 

I think I may have exacerbated my rate limiting problem by realizing quite late that I was hitting 403s repeatedly.